contracts/access/AccessControl.sol

```diff
@@ -190,10 +190,13 @@ abstract contract AccessControl is Context, IAccessControl, ERC165 {
      *
      * May emit a {RoleGranted} event.
      */
-    function _grantRole(bytes32 role, address account) internal virtual {
+    function _grantRole(bytes32 role, address account) internal virtual returns (bool) {
```

Comment:
This function (and `_revokeRole`) needs to mention what the return value means in the NatSpec. I can't make a suggestion more than 3 lines further than here but I'd propose:

```solidity
* @dev Attempts to grant `role` to `account` and returns a boolean indicating if `role` was granted.
```